app/channel.go

```diff
@@ -767,13 +767,20 @@ func (a *App) UpdateChannelMemberNotifyProps(data map[string]string, channelId s
 
 func (a *App) DeleteChannel(channel *model.Channel, userId string) *model.AppError {
 	ihc := make(chan store.StoreResult, 1)
-	ohc := a.Srv.Store.Webhook().GetOutgoingByChannel(channel.Id, -1, -1)
+	ohc := make(chan store.StoreResult, 1)
+
 	go func() {
 		webhooks, err := a.Srv.Store.Webhook().GetIncomingByChannel(channel.Id)
 		ihc <- store.StoreResult{Data: webhooks, Err: err}
 		close(ihc)
 	}()
 
+	go func(){
```

Comment:
Tiny gofmt error here. Probably would be interesting for you to configure your editor to execute gofmt after each edtion, or even better goimports (which us gofmt under the hood, so you don't need to gofmt too).

Reply:
yeah. I have it in editor, but had to click button to run go fmt/goimports command.